lib/chef/resource/execute.rb

```diff
@@ -55,28 +55,53 @@ def command(arg = nil)
       property :default_env, [ TrueClass, FalseClass ], desired_state: false, default: false,
-        description: "When true this enables ENV magic to add path_sanity to the PATH and force the locale to English+UTF-8 for parsing output"
-      property :returns, [ Integer, Array ], default: 0
-      property :timeout, [ Integer, Float ]
-      property :user, [ String, Integer ]
-      property :domain, String
-      property :password, String, sensitive: true
+               introduced: "14.2",
+               description: "When true this enables ENV magic to add path_sanity to the PATH and force the locale to English+UTF-8 for parsing output"
+
+      property :returns, [ Integer, Array ], default: 0,
+               description: "The return value for a command. This may be an array of accepted values. An exception is raised when the return value(s) do not match."
+
+      property :timeout, [ Integer, Float ],
+               description: "The amount of time (in seconds) a command is to wait before timing out."
+
+      property :user, [ String, Integer ],
+               description: "The user name of the user identity with which to launch the new process. Default value: nil. The user name may optionally be specifed with a domain, i.e. domainuser or user@my.dns.domain.com via Universal Principal Name (UPN)format. It can also be specified without a domain simply as user if the domain is instead specified using the domain attribute. On Windows only, if this property is specified, the password property must be specified."
+
+      property :domain, String,
+               introduced: "12.21",
+               description: "Windows only: The domain of the user user specified by the user property. Default value: nil. If not specified, the user name and password specified by the user and password properties will be used to resolve that user against the domain in which the system running Chef client is joined, or if that system is not joined to a domain it will resolve the user as a local account on that system. An alternative way to specify the domain is to leave this property unspecified and specify the domain as part of the user property."
+
+      property :password, String, sensitive: true,
+               introduced: "12.21",
+               description: "Windows only: The password of the user specified by the user property. Default value: nil. This property is mandatory if user is specified on Windows and may only be specified if user is specified. The sensitive property for this resource will automatically be set to true if password is specified."
 
       # lazy used to set default value of sensitive to true if password is set
       property :sensitive, [ TrueClass, FalseClass ], default: lazy { |r| r.password ? true : false }
 
-      property :elevated, [ TrueClass, FalseClass ], default: false
+      property :elevated, [ TrueClass, FalseClass ], default: false,
```

Comment:
Does this require a description?

Reply:
This was missing from the docs site. I created an issue for some of the missing docs